ocd_backend/models/model.py

```diff
@@ -195,6 +195,10 @@ def properties(self, props=True, rels=True):
         """ Returns namespaced properties with their inflated values """
         props_list = list()
         for name, prop in iterate({k: v for k, v in self.values.items() if k[0:1] != '_'}):
+            # Skip properties that are not part of the model definitions
+            if name in ('https://argu.co/ns/meeting/score', 'https://argu.co/ns/meeting/tag', '@id', '@type'):
```

Comment:
In the future, properties might be added that are not part of the model definition. Does the code break when this happens?

Reply:
Most likely yes, but I don't see a way to prevent that for properties that may be added in the future.